.github/workflows/gradle-reusable-test.yaml

```diff
@@ -64,6 +64,17 @@ jobs:
           develocity-url: https://develocity.apache.org
           develocity-plugin-version: 3.18.2
 
+      - name: Test log4j-samples-gradle-metadata
+        id: build
+        shell: bash
+        env:
+          LOG4J_VERSION: ${{ inputs.log4j-version }}
+          LOG4J_REPOSITORY_URL: ${{ inputs.log4j-repository-url }}
+        run: |
+          ./gradlew \
+          -Plog4j.version=$LOG4J_VERSION \
+          -Plog4j.repository.url=$LOG4J_REPOSITORY_URL \
```

Comment:
I guess this line is no longer needed, since `settings.gradle` accesses the `LOG4J_REPOSITORY_URL` environment variable directly. Am I right?

Reply:
The property is used in `settings.gradle` as I now kept it as it was:
https://github.com/apache/logging-log4j-samples/blob/bf205b7c0bf54897c98fab19a3e9847266cb560b/settings.gradle#L39

I would keep it like this. Then it is consistent how the properties are passed in.